cpp/openlocationcode.cc

```diff
@@ -9,31 +10,40 @@
 namespace openlocationcode {
 namespace internal {
 const char kSeparator = '+';
-const size_t kSeparatorPosition = 8;
-// The max number of digits returned in a plus code. Roughly 1 x 0.5 cm.
-const size_t kMaximumDigitCount = 15;
 const char kPaddingCharacter = '0';
 const char kAlphabet[] = "23456789CFGHJMPQRVWX";
-// Lookup table of the alphabet positions of characters 'C' through 'X',
-// inclusive. A value of -1 means the character isn't part of the alphabet.
-const int kPositionLUT['X' - 'C' + 1] = { 8, -1, -1, 9, 10, 11, -1, 12, -1, -1,
-    13, -1, -1, 14, 15, 16, -1, -1, -1, 17, 18, 19 };
+// Number of digits in the alphabet.
 const size_t kEncodingBase = 20;
+// The max number of digits returned in a plus code. Roughly 1 x 0.5 cm.
+const size_t kMaximumDigitCount = 15;
 const size_t kPairCodeLength = 10;
+const size_t kGridCodeLength = kMaximumDigitCount - kPairCodeLength;
 const size_t kGridColumns = 4;
 const size_t kGridRows = kEncodingBase / kGridColumns;
+const size_t kSeparatorPosition = 8;
 // Work out the encoding base exponent necessary to represent 360 degrees.
 const size_t kInitialExponent = floor(log(360) / log(kEncodingBase));
 // Work out the enclosing resolution (in degrees) for the grid algorithm.
 const double kGridSizeDegrees =
     1 / pow(kEncodingBase, kPairCodeLength / 2 - (kInitialExponent + 1));
-
+// Inverse (1/) of the precision of the final pair digits in degrees. (20^3)
+const size_t kPairPrecisionInverse = 8000;
+// Inverse (1/) of the precision of the final grid digits in degrees.
+// (Latitude and longitude are different.)
+const size_t kGridLatPrecisionInverse = 8000 * pow(kGridRows, kGridCodeLength);
```

Comment:
Can we reference kPairPrecisionInverse here and below to be more clear?

Reply:
Done!